### ISSUE TITLE
Query to listUsers does not return standard attributes

### ISSUE THREAD

**Issue by wolfgangmeyers** [aws/aws-sdk-js#3245]
Hello,

I am posting a new bug because the original one was locked, and I can't comment on it or reopen. The original can be found here:

https://github.com/aws/aws-sdk-js/issues/2570

It should not have been closed, because the solution mentioned in the original bug was a workaround, not a real fix. We should be able to query for a subset of the standard attributes.

**Comment by ajredniwja** [comment]
hey @wolfgangmeyers thank-you for reaching out to us with your issue, 

I think the solution mentioned in the post was valid, I would not classify it as a bug but more of an enhancement, marking it a feature request.

**Comment by wolfgangmeyers** [comment]
The docs for this api say:
```
AttributesToGet
An array of strings, where each string is the name of a user attribute to be returned for each user in the search results. If the array is null, all attributes are returned.

Type: Array of strings

Length Constraints: Minimum length of 1. Maximum length of 32.

Pattern: [\p{L}\p{M}\p{S}\p{N}\p{P}]+

Required: No
```

I don't see where it specifies "If you explicitly request any of the standard attributes, an error will be returned instead of the data you requested". I think that I was using the API correctly, and this should be considered a bug.

**Comment by ajredniwja** [comment]
Hey @wolfgangmeyers if you mark AttributesToGet to null, it will return all of them.
For AttributesToGet, it is there to get specific attributes only, I think that is the purpose of the param. Let me know if you think differently. 

Because mentioning at-least one param for example email would not result in any errors.

**Comment by wolfgangmeyers** [comment]
You're right, I was able to make the call when only specifying email. It was when I started specifying additional attributes such as given_name and family_name that I encountered the error. If it helps, I can do some testing to identify which combinations of attributes work and which do not.

**Comment by ajredniwja** [comment]
I would really appreciate if you could leave a comment on that here for future users.
